core/variant/typed_array.h

```diff
@@ -31,8 +31,10 @@
 #ifndef TYPED_ARRAY_H
 #define TYPED_ARRAY_H
 
+#include "core/object/object.h"
```

Comment:
Note for later (needs discussion), this seems needed only for `PropertyInfo` and related stuff, we might want to check if it would be useful to split property stuff out of `object.h` (e.g. `core/object/property_info.h`).